src/ModelTraits/UploadableFile.php

```diff
@@ -21,14 +28,21 @@ public static function bootUploadableFile()
     }
 
     /**
-     * Put value on desired model image attribute
+     * Put value on desired model document attribute
      *
      * @param string $fileAttributeName
      * @param string $path
      */
     public function fillUploadedFileAttributeValue(string $fileAttributeName, string $path)
     {
-        $this->{$fileAttributeName} = $path;
+        if (method_exists($this, 'isTranslatableAttribute')
+            && is_callable([$this, 'isTranslatableAttribute'])
```

Comment:
```php
@if (method_exists($crud->model, 'isTranslatableAttribute') && $crud->model->isTranslatableAttribute($field['name']) && config('backpack.crud.show_translatable_field_icon'))
<i class="fa fa-flag-checkered pull-{{ config('backpack.crud.translatable_field_icon_position') }}" style="margin-top: 3px;" title="This field is translatable."></i>
@endif
```
apparently if this method is protected, the whole application will crash... Backpack expects this method to be public